analytics-webapps/src/main/webapp/skin/less/analytics.less

```diff
@@ -64,6 +64,24 @@
   }
 }
 
+.cardDatePicker {
+  width: 400px;
+}
+.rageDatePicker {
+  width: 22px!important;
+}
+.footer{
+  background-color: white;
+  max-width: fit-content;
+}
+#analyticsDatePicker {
+  .v-text-field__slot{
+    > input {
+      padding: 0px !important;
+    }
+  }
+}
+
```

Comment:
Add those CSS rules under the parent id of CSS Class of the Analytics Portlets where you want to apply it